template/template.yaml

```diff
@@ -1,4 +1,9 @@
 options:
+  - !Option
+    name: unstable-hal
+    display_name: Enable unstable HAL features.
+    help: esp-hal contains many unstable features that are not yet ready for general use. This configuration enables using them.
```

Comment:
Maybe "not yet ready for general use" is a bit to frightening? I imagine people don't enabling it just out of fear and wondering why they can do much with esp-hal and friends?

Reply:
still not convinced by the wording but have no good suggestion at hand

Reply:
I agree that we have to make it clear that enabling unstable deps can and will force them to change their code, but I do also agree that this current wording is a bit harsh. Could we re write this to emphasize that they'll need to adapt they're code for each release instead?

I would say that crates like esp-alloc are ready functionality wise (i.e, it's not fundamentally broken), but we just haven't nailed down a stable API.

Reply:
Now we're using the same wording as `instability`.